src/base/bittorrent/session.cpp

```diff
@@ -1390,17 +1390,19 @@ void Session::configurePeerClasses()
     f.add_rule(lt::address_v4::any()
                , lt::address_v4::broadcast()
                , 1 << lt::session::global_peer_class_id);
-#if TORRENT_USE_IPV6
+
+#if (LIBTORRENT_VERSION_NUM >= 10200) || TORRENT_USE_IPV6
```

Comment:
I would define a macro and use it here:
```c++
#if (LIBTORRENT_VERSION_NUM >= 10200) || TORRENT_USE_IPV6
#define LT_USE_IPV6
#endif
```

Reply:
I think I won't prefer it because we (possibly me only) *might* forget to remove `LT_USE_IPV6` leftovers when libtorrent 1.2 is the requirement causing another regression... or do you insist?

Reply:
> or do you insist?

No.